Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/CertificateStateHelper.kt

```diff
@@ -61,10 +63,10 @@ fun IncludeCertificateQrcodeCardBinding.bindValidityViews(
     }
     when (certificate.displayedState()) {
         is CwaCovidCertificate.State.ExpiringSoon -> {
-            expirationStatusIcon.isVisible = true
+            expirationStatusIcon.isVisible = badgeCount == null || badgeCount == 0
             (expirationStatusIcon.layoutParams as ConstraintLayout.LayoutParams).verticalBias = 0f
             expirationStatusIcon.setImageDrawable(context.getDrawableCompat(R.drawable.ic_av_timer))
-            expirationStatusText.isVisible = true
+            expirationStatusText.isVisible = badgeCount == null || badgeCount == 0
```

Comment:
According to Figma expiring_soon does not have a text anymore 
1- https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.6?node-id=29791%3A32077
2- https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8718?focusedCommentId=1201339&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1201339

Reply:
Ah yes, I'll remove this.